test/jsonp_test.rb

```diff
@@ -3,7 +3,7 @@
 class JSONPControllerTester < ActionController::Base
   include Charcoal::JSONP
 
-  caches_action :test
+  caches_action :test if Rails::VERSION::MAJOR < 5
```

Comment:
Maybe wait. Somebody just did a PR to port the gem to Rails 5 when it was thought the gem wouldn't be supported ...